preview-docs/OperationsCatalog.md

```diff
@@ -0,0 +1,25 @@
+# Operations Catalog
+
+## ⚠️ BEFORE YOU START: IMPORTANT DISCLAIMER
+
+>The documentation here is all speculative, future looking and subject to change. It should not be taken as any sort of product commitment and only being shared to gather early feedback on what is on our minds. 
+
+# Why an operations catalog?
+
+Our long term mission has been to empower organizations with a developer toolkit that enables them to move fast allowing them to keep up with the competition and meet their customers’ expectations. We do this by advocating for a unified data graph and providing the means to do so, so that developers can get universal access to fragmented and disparate data and services, without needing to understand the underlying architecture and the role of each service. 
+
+As part of this developer toolkit, we provide Apollo Studio Explorer, where developers can create, run and manage operations executed against the graph. _A critical piece of this experience is the ability to save, organize and share operations, which enables cross team collaboration._
+
+Following are some of the use cases that we are considering for solutioning operations catalog in Apollo Studio. We would love to get your feedback on whether these will be useful for your organization and how they might fit in graph exploration, collaboration and consumption experience. If you're reading this, please fill out our feedback form! We are grateful for your feedback and read every response that comes in: (insert feedback form link). 
+
+# Proposed use cases
+
+1. **Save and organize operations as collections:** A collection is a set of operations. Developers will be able to save the operations that they write in Studio for later use and easy access and organize them as collections. They can create, update, delete and read the collections of operations. Collections will have documentation that can be used to explain their purpose. Comments on collections for collaboration will also be made available. 
+2. **Share with other members of the team:** Developers can share the collection/s of operations with other members of the team that have at least read access to the graph. For public graphs, sharing can be possible outside of the members of the Studio Org. Sharing of collections is made possible by:
+    1. Share the collection directly with other members of the org that have access to Studio: The shared collection will be visible in the operations catalog of each member that it is shared with. Any user with the permission to edit the collection can make changes to the collection or any of the operation. Sharing can also be done as view-only, where the user with view-only permission cannot make any changes to the collection.
```

Comment:
Do you provide permission on the collection on a user basis? That may be a rabbit hole with respect to database modeling. Could we start with the simpler approach of "graph admins+ can edit all collections" or something?

Reply:
ohh good point. Yes love the modification.